conviva/README.md

```diff
@@ -26,26 +26,28 @@ This monitor requires:
 
 ### INSTALLATION
 
-Install the latest version of the SignalFx Smart Agent as described [here](https://github.com/signalfx/integrations/tree/master/signalfx-agent).
+Install the latest version of the SignalFx Smart Agent as described in [Installation](https://github.com/signalfx/integrations/tree/master/signalfx-agent).
```

Comment:
Since we have a Smart Agent tile, instead of linking to GitHub, let's link back to the Smart Agent tile. I can't seem to find an example of one tile linking to the agent tile, but it is possible. For instance, this is the markup for linking from the Google App Engine tile to the Google Cloud Platform tile. We may need to ask around or look at other tiles, but this is example: 

> Use SignalFx to monitor Google App Engine via [Google Cloud Platform](https://github.com/signalfx/integrations/tree/master/gcp)[](sfx_link:gcp).